src/zenml/integrations/slack/alerters/slack_alerter.py

```diff
@@ -253,64 +280,76 @@ def post(
             response = client.chat_postMessage(
                 channel=slack_channel_id, text=message, blocks=blocks
             )
+            if not response["ok"]:
+                logger.error("Failed to send message to Slack channel.")
```

Comment:
This should log why it failed to help debug this, otherwise the user has no clue what failed

Reply:
Fixed this.